### ISSUE TITLE
Export function should be on file menu

### ISSUE THREAD

**Issue by webjoyable** [tldraw/tldraw-beta#53]
I accidentally stumbled upon the export button, this really should be in the File menu. Also, when exporting a drawing, and nothing is specifically selected, the filename should be equal to that page name

Just a suggestion

**Comment by TodePond** [comment]
That's a good point!

**Comment by nielsbom** [comment]
I totally agree. Most of all because exporting is now under "Edit" and I don't think exporting is an edit-kind-of operation.
